Sources/Logging/Logging.swift

```diff
@@ -694,18 +706,6 @@ public enum LoggingSystem {
         }, validate: true)
     }
 
-    /// `bootstrap` is a one-time configuration function which globally selects the desired logging backend
-    /// implementation. `bootstrap` can be called at maximum once in any given program, calling it more than once will
-    /// lead to undefined behavior, most likely a crash.
-    ///
-    /// - parameters:
-    ///     - factory: A closure that given a `Logger` identifier, produces an instance of the `LogHandler`.
-    public static func bootstrap(_ factory: @escaping (String) -> LogHandler) {
```

Comment:
this just moved "up" a little in the file, to be organized together with the other ones